sync_diff_inspector/utils/utils.go

```diff
@@ -1043,3 +1043,7 @@ func IsRangeTrivial(rangeCond string) bool {
 	}
 	return strings.ToLower(rangeCond) == "true"
 }
+
+func IsBinaryColumn(col *model.ColumnInfo) bool {
+	return col.GetType() == mysql.TypeString && mysql.HasBinaryFlag(col.GetFlag())
```

Comment:
does `varbinary` has type `TypeVarString`？

Reply:
The unit test should cover this check.

Reply:
fix it